<launch>
  <arg name="positions_file" value="$(find riptide_controllers)/cfg/vehicle_properties.yaml" />
  <!-- Do I need to find device here? -->
  
  <!-- <remap from > -->
  <include file="$(find nortek_dvl)/launch/dvl.launch" />
  <node pkg="riptide_hardware" type="dvl_processor" name="dvl_processor" output="screen">
    <param name="properties_file" type="string" value="$(arg positions_file)" />
  </node>
</launch>